distributions/rhel-84.json

```diff
@@ -12,6 +12,9 @@
     }, {
       "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8/x86_64/appstream/os",
       "rhsm": true
+    }, {                                                                                                                                                      
+      "baseurl": "https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/ansible/2.9/os",                                                                   
+      "rhsm": true                                                                                                                                            
```

Comment:
How about we add a field here `"requires_subscription": [ 'SKUxxxxx', 'ansible', etc... ]`. If present image-builder will check if the logged in user has access to these, and adds the packages (which should be located in a separate file), when querying the `/packages` endpoint. That way the user can search for/select them in the UI.

 The separate packages file should have a predictable name I think. Something like

`rhel-84-x86_64-packages.$SKU.json` with $SKU corresponding to the subscription name.

I'm not sure if we'll use SKU numbers or a name (for example `ansible`), whatever is easiest, both should be treated as strings anyway.